helm/wfl/Chart.yaml

```diff
@@ -4,10 +4,10 @@ description: A Helm chart for deploy WorkFlow Launcher on Kubernetes
 
 type: application
 
-version: 0.2.6
+version: 0.2.7
 
 home: https://github.com/broadinstitute/wfl
-appVersion: 0.2.6
+appVersion: 0.2.7
```

Comment:
Even though my change to values.yaml is a no-op, the linting wants a version bump here. If we want, we can revert afa751a to get rid of this version bump (linting will complain but it won't affect anything).